templates/taxi/car_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} Сar</h1>
+
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+    {% if object %}
+      <a class="btn btn-primary" href="{% url 'taxi:car-detail' pk=car.id %}">Cancel</a>

```

Comment:
The 'Cancel' button should lead to the previous page the user was on, not necessarily the car detail page. Consider using JavaScript to go back to the previous page.